idunn/blocks/delivery.py

```diff
@@ -0,0 +1,32 @@
+from enum import Enum
+from typing import List, Literal
+
+from .base import BaseBlock
+
+
+class DeliveryType(Enum):
+    CLICK_AND_COLLECT = "click_and_collect"
+    DELIVERY = "delivery"
+    TAKEAWAY = "takeaway"
+
+
+class DeliveryBlock(BaseBlock):
+    type: Literal["delivery"] = "delivery"
+    available: List[DeliveryType]
```

Comment:
Any specific reason to use a list here ?  

Explicit fields `click_and_collect`, `delivery`, `takeway` could be easier to maintain and more discoverable.   
Similarly to the `AccessibilityBlock`, the possible values would be `"yes"` or `"unknown"`.

Reply:
I had no strong opinion about both options so I'll just trust yours (plus "we already do it the over way" seems like a good argument).

Reply:
Now it looks like that:

```json
{
    "type": "delivery",
    "click_and_collect": "yes",
    "takeaway": "unknown",
    "delivery": "yes"
}
```